### PR TITLE
add contributors to README.md

### PR DESCRIPTION
**Pull request by kentwills**
Since history was wiped I wanted to make sure these contributors are mentioned on the front page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,23 @@ Beans is an internal networking tool to help employees connect with each other a
...
 * [Yelp Beans Wiki](https://github.com/Yelp/beans/wiki)
 
+## Original Contributors
+* Clare Curtis
+* Matt Grounds
+* Alex Levin
+* Xi Li
+* Aaron Loo
+* Bryan Marty
+* Andrew Mason
```
Comment (ajm188) [inline README.md]: [Andrew Mason](https://github.com/ajm188) etc

Comment (ajm188) [inline README.md]: Or let people link as they want, as this could be a giant pain.

Comment (kentwills) [inline README.md]: Yeah I am for people linking after the fact.

Comment (ajm188) [inline README.md]: ack merge it

**Pull request merged**